src/test/java/baseball/util/GameInputValidatorTest.java

```diff
@@ -0,0 +1,75 @@
+package baseball.util;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+
+class GameInputValidatorTest {
+    private GameInputValidator gameInputValidator;
+
+    @BeforeEach
+    public void setUp() {
+        gameInputValidator = new GameInputValidator();
+    }
+
+    @Test
+    @DisplayName("inputNum이 서로 다른 숫자가 아닌 경우")
+    void 서로_다른_수가_아닌_경우() throws Exception {
+        //given
+        String inputNum = "122";
+        //when
+        //then
+        assertThatIllegalArgumentException()
+                .isThrownBy(() -> gameInputValidator.validate(inputNum))
+                .withMessage(ErrorMessage.SAME_INPUT_NUM.getErrorMessage());
+    }
+
+    @Test
+    @DisplayName("두자리가 입력된 경우")
+    void 두자리_수가_입력된_경우() throws Exception {
+        //given
+        String inputNum = "12";
+        //when
+        //then
+        assertThatIllegalArgumentException()
+                .isThrownBy(() -> gameInputValidator.validate(inputNum))
+                .withMessage(ErrorMessage.INVALID_GAME_INPUT_NUM.getErrorMessage());
+    }
+
+    @Test
+    @DisplayName("네자리가 입력된 경우")
+    void 네자리_수가_입력된_경우() throws Exception {
+        //given
+        String inputNum = "1234";
+        //when
+        //then
+        assertThatIllegalArgumentException()
+                .isThrownBy(() -> gameInputValidator.validate(inputNum))
+                .withMessage(ErrorMessage.INVALID_GAME_INPUT_NUM.getErrorMessage());
+    }
+
```

Comment:
`@ParamterizedTest`를 사용하면 자리 수 관련해서 공통적인 입력을 만들어서 테스트할 수 있을 것 같아요!
```suggestion
    @ParameterizedTest
    @ValueSource(strings={"12","1234"})
    @DisplayName("[2,4]자리가 입력된 경우")
    void 두자리_수가_입력된_경우(String inputNum) throws Exception {
        //given
    
        //when
        //then
        assertThatIllegalArgumentException()
                .isThrownBy(() -> gameInputValidator.validate(inputNum))
                .withMessage(ErrorMessage.INVALID_GAME_INPUT_NUM.getErrorMessage());
    }
```